packages/alpinejs/src/utils/on.js

```diff
@@ -93,6 +93,8 @@ function isNumeric(subject){
 }
 
 function kebabCase(subject) {
+    if ([' ','_'].includes(subject
+    )) return subject
```

Comment:
If that is the intended behaviour, wouldn't you just remove `.replace(/[_\s]/, '-')` from the next line instead?

Reply:
Then it won't kebab case strings.

I'm not positive what the kebabcase is doing here (maybe I can't think up a good reason it exists?).

So I elected to not remove something I didn't understand the use-case for.

I would assume that removing the kebabcase part entirely would work.

Reply:
if there are spaces or underscores, they will never hit the last line because of your addition making that instruction redundant anyway, though.
If `Arrow Right` needs to be converted to `arrow-right` then your addition would break things as well.

Unfortunately I don't have a good reference of key names on all browser, but that should probably be looked at before introducing this change.

Reply:
No it won't, this will only catch ` ` and `_` not anything that includes those.

I think you're reading the flow backwards.

This is equivalent to `if (subject === ' ' || subject === '_')`

Reply:
Ah, right. You are looking for exact matches. Fair enough, just ignore my comment. 😅